### PR TITLE
Enable tests in R CMD CHECK

### PR DESCRIPTION
**Pull request by grst**
Actually, the tests were never run because of a missing line in `testthat.R`

### CODE REVIEW

**Comment by LorenzoMerotto** [comment]
@grst I just finished updating the branch with my additions and it seems that all the tests work. I haven't performed the merge yet. 

If the tests you run are the same ones that I had in my branch I will merge my PR, and wait for you to re trigger the tests

**Comment by grst** [comment]
The checks work there, but the tests did not run, because [this line](https://github.com/omnideconv/omnideconv/pull/123/files#diff-9fab7e874754e477463ce086b5db508fd76d8a2f55451c5534e4f282f8de6b41) is missing. 

You would need to add that to your branch, or we first make the tests pass here and then update your branch accordingly.

**Comment by LorenzoMerotto** [comment]
@alex-d13 seems it worked

File: tests/testthat/test-b-buildmodel.R [hunk 1]
```diff
@@ -65,7 +65,7 @@ test_that("DWLS build signature matrix works", {
 })
 
 test_that("CIBERSORTx build signature matrix works", {
-  set_cibersortx_credentials(Sys.getenv("CIBERSORTX_EMAIL"), Sys.getenv("CIBERSORTX_TOKEN"))
+  set_cibersortx_credentials("lorenzo.merotto@studenti.unipd.it", "721a387e91c495174066462484674cb8")
```
Comment (grst) [inline tests/testthat/test-b-buildmodel.R:68]: This should get the yoken from the repo secrets

File: tests/testthat/test-c-deconvolution.R [hunk 2]
```diff
@@ -175,7 +175,7 @@ test_that("DWLS deconvolution works", {
 
 
 test_that("CIBERSORTx deconvolution works", {
-  set_cibersortx_credentials(Sys.getenv("CIBERSORTX_EMAIL"), Sys.getenv("CIBERSORTX_TOKEN"))
+  set_cibersortx_credentials("lorenzo.merotto@studenti.unipd.it", "721a387e91c495174066462484674cb8")
```
Comment (grst) [inline tests/testthat/test-c-deconvolution.R:178]: Same here

**Review by grst — COMMENTED** [review]
Nice!

File: tests/testthat/test-b-buildmodel.R (hunk 1 above)
Comment (LorenzoMerotto) [reply tests/testthat/test-b-buildmodel.R:68]: I'll try that out!

**Review by LorenzoMerotto — APPROVED** [review]

File: tests/testthat/test-b-buildmodel.R (hunk 1 above)
Comment (LorenzoMerotto) [reply tests/testthat/test-b-buildmodel.R:68]: How would you implement it? I don't really know where to store the credentials

**Comment by grst** [comment]
Just revert that change. They should be available in those environment
variables.

If it doesn't work anymore, the token may have expired. You can set the new
one in the repository settings -> secrets -> actions

**Comment by grst** [comment]
I forgot something... see my last commit.

Secrets are not directly available as env vars, but it's possible to make them available.

**Comment by LorenzoMerotto** [comment]
@grst I see that the tests are still failing. Should I store my cibersort credentials somewhere?

**Comment by LorenzoMerotto** [comment]
@grst I guess we can merge now, right?

**Comment by grst** [comment]
Sure, go ahead!

**Comment by grst** [comment]
Sure, go ahead!

**Pull request merged**